### ISSUE TITLE
Can I achieve this type of header animation with this library?

### ISSUE THREAD

**Issue by tayguohong** [benevbright/react-navigation-collapsible#128]
initial state:
<img width="370" alt="Screenshot 2020-04-28 at 2 45 03 PM" src="https://user-images.githubusercontent.com/57252173/80456291-e8de4280-895f-11ea-9d64-85dcf04a19fd.png">
After scrolling until specific height:
<img width="377" alt="Screenshot 2020-04-28 at 2 45 33 PM" src="https://user-images.githubusercontent.com/57252173/80456376-0f03e280-8960-11ea-8577-5e9105b4d144.png">

**Comment by benevbright** [comment]
No, and I don't think this module can support it anytime soon but I will write a comment here when it's possible.

**Comment by tayguohong** [comment]
> No, and I don't think this module can support it anytime soon but I will write a comment here when it's possible.

Alright, thank you for fast reply and this library too.

**Comment by benevbright** [comment]
I opened https://github.com/benevbright/react-navigation-collapsible/pull/159 to resolve this issue. (WIP)
There is still one blocker. On Android, react-navigation's `backgroundHeader` is non-touchable. I may open a PR there and see how they think about it.
